lab09/lab_09.cpp

```diff
@@ -174,7 +203,16 @@ void send_to_output_high_to_low(double fine_cost[], string patron_name[], int am
   //outputs all values sorted from the highest fine cost to the lowest
   for(unsigned current_index = INITIAL_INDEX; current_index < amount_of_records; current_index++)
   {
-  output_file << patron_name[current_index] << setfill(' ') << setw(33) << "$ " << fine_cost[current_index] << fixed << setprecision(2) << endl;
+    output_file
+      << patron_name[current_index]
+      << setfill(' ') << setw(31)
+      << "$ ";
+
+    output_file
+      // Format the decimal, fixed, 2 decimals, left pad.
+      << fixed << setprecision(2) << setfill(' ') << setw(5)
+      << fine_cost[current_index]
+      << endl;

```

Comment:
Did a little formatting for readability.

Fixed a bug where your precision came after output (at least on my compiler).  Also left-padded the amounts. for consistent decimal placement in the file.